benchmarks/multimodal/sweep/experiments/epd/run_experiment.py

```
@@ -0,0 +1,639 @@
+    return parser
+
+
+class ImageServer(http.server.ThreadingHTTPServer):
+    request_queue_size = 128
+    daemon_threads = True
+
+
+class QuietHandler(http.server.SimpleHTTPRequestHandler):
+    protocol_version = "HTTP/1.1"
+
+    def log_message(self, format: str, *args: object) -> None:
```

Ruff (0.16.2)

[error] Function argument `format` is shadowing a Python builtin

(A002)